tests/functional/src/main/java/org/mule/tck/junit4/ApplicationContextBuilder.java

```diff
@@ -89,4 +93,9 @@ protected void configureMuleContext(MuleContextBuilder contextBuilder)
         contextBuilder.setWorkListener(new TestingWorkListener());
     }
 
+    public void disableMuleContextStart()
```

Comment:
setters is a bad idea for this because it can be called at any time but is only useful when used before build. Overload the build method instead